### DISCUSSION TITLE
How to Display Steam Now Playing With Next.js — Jagad Yudha Awali

### DISCUSSION THREAD

**Discussion by giscus** [jagadyudha/jagad.dev#97] [Announcements]
# How to Display Steam Now Playing With Next.js — Jagad Yudha Awali

Using Steam web API to display now playing steam on websites

https://jagad.dev/posts/steam-now-playing

**Comment by freakend** [comment]
I love your tutorial! but how about the recent game you showed in the activities page? looking forward to read the tutorial about that, too :p

**Reply by jagadyudha** [reply]
You cand read API documention [here](https://partner.steamgames.com/doc/webapi/IPlayerService#GetRecentlyPlayedGames) and put your steam id and access token in this url: 
```https://api.steampowered.com/IPlayerService/GetRecentlyPlayedGames/v1/?access_token={YOUR_ACCESS_TOKEN}&steamid={STEAMID}```
